langserve/server.py

```diff
@@ -325,6 +333,7 @@ def add_routes(
     output_type: Union[Type, Literal["auto"], BaseModel] = "auto",
     config_keys: Sequence[str] = (),
     include_callback_events: bool = False,
+    per_req_config_modifier: PerRequestConfigModifier = lambda config, _: config,
```

Comment:
nit: Instead of identity, could we set it to None and use an optional type? I find `None` to be easier to interpret -- i.e., there's No per request modification applied